tests/page/page-request-intercept.spec.ts

```diff
@@ -199,3 +199,64 @@ it('should fulfill after redirects', async ({page, server}) => {
   expect(response.headers()['content-type']).toBe('text/plain');
   expect(await response.text()).toBe('Yo, page!');
 });
+
+it('should fulfill original response after redirects', async ({page, browserName, server}) => {
+  it.fixme(browserName !== 'chromium');
+  server.setRedirect('/redirect/1.html', '/redirect/2.html');
+  server.setRedirect('/redirect/2.html', '/title.html');
+  const expectedUrls = ['/redirect/1.html', '/redirect/2.html', '/title.html'].map(s => server.PREFIX + s);
+  const requestUrls = [];
+  const responseUrls = [];
+  const requestFinishedUrls = [];
+  page.on('request', request => requestUrls.push(request.url()));
+  page.on('response', response => responseUrls.push(response.url()));
+  page.on('requestfinished', request => requestFinishedUrls.push(request.url()));
+  await page.route('**/*', async route => {
```

Comment:
Let's count the number of route handler calls - that should be one, right?

Reply:
Added.